Formula/hashcat.rb

```diff
@@ -22,9 +22,23 @@ class Hashcat < Formula
   end
 
   depends_on "gnu-sed" => :build
+  depends_on "minizip"
+  depends_on "xxhash"
+
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "opencl-headers" => :build
+    depends_on "ocl-icd"
+    depends_on "pocl"
+  end
 
   def install
-    system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}"
+    system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}",
+      "USE_SYSTEM_XXHASH=1",
+      "USE_SYSTEM_OPENCL=1",
+      "USE_SYSTEM_ZLIB=1",
+      "ENABLE_UNRAR=0"
     system "make", "install", "CC=#{ENV.cc}", "PREFIX=#{prefix}"
```

Comment:
Something like following otherwise final linkage seems to be wrong.
```suggestion
    args = %W[
      CC=#{ENV.cc}
      PREFIX=#{prefix}
      USE_SYSTEM_XXHASH=1
      USE_SYSTEM_OPENCL=1
      USE_SYSTEM_ZLIB=1
      ENABLE_UNRAR=0
    ]
    system "make", *args
    system "make", "install", *args
```